src/test/kotlin/dev/stalla/builder/validating/podcast/ValidatingPodcastPodcastindexBuilderTest.kt

```diff
@@ -29,92 +33,166 @@ internal class ValidatingPodcastPodcastindexBuilderTest {
+            assertThat(podcastPodcastindexBuilder).prop(PodcastPodcastindexBuilder::hasEnoughDataToBuild).isFalse()
 
-            assertThat(podcastPodcastBuilder.build()).isNull()
+            assertThat(podcastPodcastindexBuilder.build()).isNull()
         }
     }
 
     @Test
     internal fun `should build a Podcast Podcastindex with at least a locked builder`() {
-        val podcastPodcastBuilder = ValidatingPodcastPodcastindexBuilder()
+        val podcastPodcastindexBuilder = ValidatingPodcastPodcastindexBuilder()
             .lockedBuilder(expectedLockedBuilder)
 
         assertAll {
-            assertThat(podcastPodcastBuilder).prop(PodcastPodcastindexBuilder::hasEnoughDataToBuild).isTrue()
+            assertThat(podcastPodcastindexBuilder).prop(PodcastPodcastindexBuilder::hasEnoughDataToBuild).isTrue()
 
-            assertThat(podcastPodcastBuilder.build()).isNotNull().prop(PodcastPodcastindex::locked).isEqualTo(expectedLockedBuilder.build())
+            assertThat(podcastPodcastindexBuilder.build()).isNotNull().prop(PodcastPodcastindex::locked).isEqualTo(expectedLockedBuilder.build())
         }
     }
 
     @Test
     internal fun `should not build a Podcast Podcastindex when there is only a locked builder that doesn't build`() {
-        val podcastPodcastBuilder = ValidatingPodcastPodcastindexBuilder()
+        val podcastPodcastindexBuilder = ValidatingPodcastPodcastindexBuilder()
             .lockedBuilder(ValidatingPodcastPodcastindexLockedBuilder())
 
         assertAll {
-            assertThat(podcastPodcastBuilder).prop(PodcastPodcastindexBuilder::hasEnoughDataToBuild).isFalse()
+            assertThat(podcastPodcastindexBuilder).prop(PodcastPodcastindexBuilder::hasEnoughDataToBuild).isFalse()
 
-            assertThat(podcastPodcastBuilder.build()).isNull()
+            assertThat(podcastPodcastindexBuilder.build()).isNull()
         }
     }
 
     @Test
-    internal fun `should build a Podcast Podcastindext with at least one funding builder`() {
-        val podcastPodcastBuilder = ValidatingPodcastPodcastindexBuilder()
+    internal fun `should build a Podcast Podcastindex with at least one funding builder`() {
+        val podcastPodcastindexBuilder = ValidatingPodcastPodcastindexBuilder()
             .addFundingBuilder(firstExpectedFundingBuilder)
 
         assertAll {
-            assertThat(podcastPodcastBuilder).prop(PodcastPodcastindexBuilder::hasEnoughDataToBuild).isTrue()
+            assertThat(podcastPodcastindexBuilder).prop(PodcastPodcastindexBuilder::hasEnoughDataToBuild).isTrue()
 
-            assertThat(podcastPodcastBuilder.build()).isNotNull()
+            assertThat(podcastPodcastindexBuilder.build()).isNotNull()
                 .prop(PodcastPodcastindex::funding).containsExactly(firstExpectedFundingBuilder.build())
         }
     }
 
     @Test
     internal fun `should not build a Podcast Podcastindex when there is only a funding builder that doesn't build`() {
```

Comment:
Improving my older test name...
```suggestion
    internal fun `should not build a Podcast Podcastindex with only a funding builder that doesn't build`() {
```